src/dispersion.rs

```diff
@@ -23,4 +23,7 @@ pub fn call() {
 	}
 
 	ret(&dispersed[..])
-}
\ No newline at end of file
+}
+
+#[no_mangle]
+pub fn deploy() { }
```

Comment:
there should not be deploy in any of those tests

with deploy, they will get packed as constructors, while they need to get packed as raw wasm module

Reply:
should be fixed now

Reply:
(reinstall pwasm-utils-cli)

Reply:
Removed all `deploy` symbols!